src/views/CGI/CGI.tsx

```diff
@@ -0,0 +1,257 @@
+import React, { useEffect, useState } from 'react'
+import { Container } from 'react-neu'
+import styled from 'styled-components'
+
+import Page from 'components/Page'
+import {
+  ProductPageHeader,
+  ProductPageContent,
+  TokenStats,
+  PriceChanges,
+  WalletBalance,
+  Description,
+  IndexComponentsTable,
+} from 'components/ProductPage'
+import { BuySellWrapper } from 'components/BuySell'
+import ExternalLink from 'components/ExternalLink'
+import MarketData from './components/MarketData'
+
+import useCgiTokenMarketData from 'hooks/useCgiTokenMarketData'
+import useCgiIndexComponents from 'hooks/useCgiIndexComponents'
+import useBalances from 'hooks/useBalances'
+
+import DpiIndexCalculationImage from 'assets/dpi-index-calculation.png'
+import { fetchHistoricalTokenMarketData } from 'utils/tokensetsApi'
+import { coingeckoCgiId } from 'constants/coingeckoIds'
+
+const CgiProductPage = (props: { title: string }) => {
+  useEffect(() => {
+    document.title = props.title
+  }, [])
+
+  const [marketCap, setMarketCap] = useState(0)
+
+  useEffect(() => {
+    fetchHistoricalTokenMarketData(coingeckoCgiId)
+      .then((response: any) => {
+        setMarketCap(response)
+      })
+      .catch((error: any) => console.log(error))
+  }, [])
```

Comment:
remove

Reply:
when we eventually use this in the future, it should come from a provider right?

Reply:
yes